recipe/meta.yaml

```diff
@@ -1,6 +1,64 @@
+{% set sdata = load_setup_py_data() %}
 {% set name = "prefect" %}
 {% set version = "0.13.10" %}
 
+
+
+
+# Maps dependency names in pip to their conda alternative
+{%
+  set name_mapping =
+  {
+    "msgpack": "msgpack-python",
+    "docker": "docker-py",
```

Comment:
These aren't the only deps that should be special cased, `dask-core` should be used instead of `dask`. There might be more.

Reply:
Of course :(

Reply:
I've fixed this one but will leave this as a reminder to compare the remaining libs